rootfs/entrypoint.sh

```diff
@@ -62,5 +62,22 @@ COMMAND="autossh "\
 
 echo "[INFO ] # ${COMMAND}"
 
-# Run command
-exec ${COMMAND}
+# Run autossh command in background and save pid
+exec ${COMMAND} &
+pid=$!
+sleep 1
```

Comment:
Why are we sleeping here?  If we're just sleeping here to give time for the tunnel to instantiate, we should find a better way to confirm that.

Also, if that fails, we should `exit()` in some fashion.